client/asyncns.c

```diff
@@ -77,7 +77,7 @@ enum {
 };
 
 struct asyncns {
-    int fds[MESSAGE_FD_MAX];
+    int fds[4];
```

Comment:
No, because they are using autoconf to define their config.h and a lot of defines.